src/main/java/com/minecolonies/coremod/sounds/BarbarianSounds.java

```diff
@@ -0,0 +1,13 @@
+package com.minecolonies.coremod.sounds;
+
+import net.minecraft.util.SoundEvent;
+
+/**
+ * Created by Asher on 12/6/17.
+ */
+public class BarbarianSounds
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1118)